downstream/modules/hub/con-enable-view-only.adoc

```diff
@@ -1,9 +1,10 @@
 // Module included in the following assemblies:
 // assembly-view-only-access.adoc
 
-
 [id="con-enable-view-only"]
 
+= Enabling view-only access for your {PrivateHubName}
+
 Enable view-only access for your private {HubName} by editing the inventory file found on your {PlatformName} installer.
```

Comment:
```suggestion
Enable view-only access for your {PrivateHubName} by editing the inventory file found on your {PlatformName} installer.
```

Reply:
Thanks, @jself-sudoku. Changes made 👍🏾

Reply:
Whoops, I missed something. I'll fix it now.